### ISSUE TITLE
Tabulator: *select* header filter can't be cleared on Firefox

### ISSUE THREAD

**Issue by maximlt** [holoviz/panel#5872]

**Comment by mayonnaisecolouredbenz7** [comment]
To expand on this, the same behavior is seen when tabulator editor is set to list.
   ```
 editors = {
        "ColumnName": {
            "type": "list",
            "values": ["Option1","Option2"],
            "clearable": True
        }
```

Works fine in chromium based browsers but clear "x" is missing in firefox

**Comment by maximlt** [comment]
> Works fine in chromium based browsers but clear "x" is missing in firefox

Indeed, that's the web browser issue reported by Tabulator's maintainer [here](https://github.com/olifolkerd/tabulator/issues/3907#issuecomment-1318215098).

**Comment by ranjithdrkumar** [comment]
Hi , the same issue happening with me on chrome as well, any quick suggestions will be very helpful.
